src/UnitySampleProject/Assets/GelatoTest.cs

```diff
@@ -0,0 +1,240 @@
+using System.Numerics;
+using System.Threading;
+using System.Threading.Tasks;
+using ChainSafe.GamingSDK.EVM.MetaMaskBrowserWallet;
+using ChainSafe.GamingWeb3.Build;
+using ChainSafe.GamingWeb3.Unity;
+using ChainSafe.GamingSdk.Gelato;
+using ChainSafe.GamingSdk.Gelato.Dto;
+using ChainSafe.GamingSdk.Gelato.Types;
+using ChainSafe.GamingWeb3;
+using Microsoft.Extensions.DependencyInjection;
+using Nethereum.Hex.HexTypes;
+using UnityEngine;
+using Web3Unity.Scripts.Library.Ethers.JsonRpc;
+using Web3Unity.Scripts.Library.Ethers.Transactions;
+using Contract = Web3Unity.Scripts.Library.Ethers.Contracts.Contract;
+
+public class GelatoTest : MonoBehaviour
```

Comment:
This one should be removed from the codebase before the release OR it can be transformed into one of the example (prefab) scripts.

Reply:
@oleksandrchainsafe @KBryan 

Yes I would prefer this to be turned into a prefab, unfamiliar with the process though

Reply:
If you have the working functionality in a scene you can just drag and drop the `GameObject` into the working directory.  This process creates the prefab. Then when someone what to integrate this. They can just drag the prefab into the scene.

Reply:
@KBryan I'm not sure what distinction is needed for working functionality in a scene. 

I have a module component I pull in, bind functions to button click events and the examples operate. How that relates to what you described I'm not sure

Reply:
Instead of having all of the elements bound in a scene. You would just have a prefab that lives in a scene.